From 2b7e5bb55311751d4915717fe640f37aaea77eed Mon Sep 17 00:00:00 2001
From: Bob Beck <bbe@google.com>
Date: Fri, 9 Jun 2023 13:24:21 -0600
Subject: [PATCH 9/9] Avoid passing NULL to memcmp

---
 net/cert/pki/general_names.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/cert/pki/general_names.cc b/net/cert/pki/general_names.cc
index bafc81ca2c065..de3b1a6503cf9 100644
--- a/net/cert/pki/general_names.cc
+++ b/net/cert/pki/general_names.cc
@@ -40,6 +40,7 @@ bool IsSuffixZero(const IPAddressBytes& mask, unsigned prefix_length) {
   size_t zero_bits = mask.size() * CHAR_BIT - prefix_length;
   size_t zero_bytes = zero_bits / CHAR_BIT;
   std::vector<uint8_t> zeros(zero_bytes, 0);
+  zeros.reserve(16);  // Reserve some data so we don't send NULL to memcmp.
   if (memcmp(zeros.data(), mask.data() + mask.size() - zero_bytes, zero_bytes))
     return false;
   size_t leftover_bits = zero_bits % CHAR_BIT;
-- 
2.41.0.162.gfafddb0af9-goog

