From de1b7ad6a06d66fc984e19d4bab57819c9ffff00 Mon Sep 17 00:00:00 2001
From: Bob Beck <bbe@google.com>
Date: Fri, 2 Jun 2023 12:24:02 +0200
Subject: [PATCH 16/17] include iterator for back_inserter

---
 net/cert/pki/crl.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/cert/pki/crl.cc b/net/cert/pki/crl.cc
index ad85f95f75403..8cc4574c9da53 100644
--- a/net/cert/pki/crl.cc
+++ b/net/cert/pki/crl.cc
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include <algorithm>
+#include <iterator>
 
 #include "net/cert/pki/cert_errors.h"
 #include "net/cert/pki/crl.h"
-- 
2.41.0.rc0.172.g3f132b7071-goog

